### PR TITLE
[he] validation for ipv4/ipv5

### PR DESCRIPTION
**Pull request by d-knafo**

### CODE REVIEW

File: src/he/validation.php [hunk 1]
```diff
@@ -47,8 +47,8 @@
     'in_array'             => 'שדה :attribute לא קיים ב:other.',
     'integer'              => 'שדה :attribute חייב להיות מספר שלם.',
     'ip'                   => 'שדה :attribute חייב להיות כתובת IP תקנית.',
-    'ipv4'                 => 'The :attribute must be a valid IPv4 address.',
-    'ipv6'                 => 'The :attribute must be a valid IPv6 address.',
+    'ipv4'                 => 'שדה :attribute חייב להיות כתובת IPv4 תקנית.',
+    'ipv6'                 => 'The :attribute חייב להיות כתובת IPv6 תקנית.',
```
Comment (caouecs) [inline src/he/validation.php]: Hello,
you forgot "The"

**Comment by d-knafo** [comment]
i fix it :)

**Comment by caouecs** [comment]
Thank you

can you translate "after_or_equal" please ? ;)

**Comment by d-knafo** [comment]
Ok :100:

**Pull request merged**